src/groups/GroupActions.js

```diff
@@ -167,7 +170,7 @@ export const makeQueryFromState = (/*type*/) => {
           commands: [
             ...matches,
             // {$sort: {'statistics.comments.all.all.count': -1}},
-            {$skip: 0},
+            {$skip: page * 20},
```

Comment:
I'm agree but what kind of logic should we apply? if we want to make the user change this setting will need UI for that.